lalrpop/src/grammar/repr.rs

```diff
@@ -60,6 +60,7 @@ pub struct Grammar {
     pub module_attributes: Vec<String>,
 }
 
+#[allow(clippy::large_enum_variant)] // TODO: verify if this is justified
```

Comment:
What were the details? Is `TypeRepr` the bigger variant? We might try to put it in a box then.